packages/components/scripts/icon-component.hbs

```diff
@@ -20,7 +20,9 @@ export class {{className}} {
   /** (optional) The width and height in pixels */
   @Prop() size?: number = 24;
   /** (optional) Sets the icon color via the `fill` attribute */
```

Comment:
Since both `color` and `fill` do the same thing, I'm thinking it might end up being confusing for maintainers, and even consumers.

I would try and document `color` strictly as an alias for `fill`. Something like:

```js
/** (optional) Sets the icon color via the `fill` attribute */
@Prop() fill?: string = 'currentColor'; 
/** (optional) Alias for `fill` */
@Prop() color?: string = 'currentColor';
```

Reply:
That looks great!

That's a good addition